CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # Changelog
 
+## v1.0.0-rc1 - Unreleased
+
+- In the `gleam/erlang/process` module:
+  - The `Name` type has been introduced. This type is used to give processes
+    names, making it easier to pass references around your application, and to
+    have a new process take over a role from a previous one that has crashed.
+  - The `new_name` function has been added for creating new names.
+  - The `named_subject` function has been added for creating a subject for a
+    given name.
+  - The `register` function now takes a name rather than an atom.
+  - The `unregister` function now takes a name rather than an atom.
+  - The `named` function now takes a name rather than an atom.
+  - The `try_call` and `try_call_forever` functions have been removed in favour
+    of `call` and `call_forever`.
+  - The `start` function has been replaced by the `spawn` and `spawn_unlinked`
+    functions.
+  - The `subject_owner` function now returns a result as a named subject may not
+    any process registered for that name.
+- In the `gleam/erlang/node` module:
+  - The `send` function now sends messages in the same format as the `send`
+    process from the `gleam/erlang/process` module.
+  - The `send` function now accepts a `Name` rather than an `Atom`.
+  - The `to_atom` function has been removed.
+- The `gleam/erlang/reference` module has been created with:
+  - The `Reference` type.
+  - The `new` function.
+- The `gleam/erlang/application` module was created with:
+  - The `priv_directory` function, formerly of the `gleam/erlang` module.
+  - The `StartType` type.
+- In the `gleam/erlang/atom` module:
+  - The `AtomNotLoaded` type has been removed.
+  - The error type of `from_string` is now `Nil`.
+  - The `from_string` function has been renamed to `get`.
+  - The `create_from_string` function has been renamed to `create`.
+- In the `gleam/erlang/node` module:
+  - The `send` function has been renamed to `untyped_send`.
+- The `gleam/erlang` module has been removed.
```

Comment:
Trying to mentally map where to reach for functionality that lived here previously, only one I see us getting consistent questions about is `get_line`. May be worth including a suggested replacement in the changelog, even if it's "include this small ffi snippet".

Reply:
I couldn't find one. My suspicion is there's a reason it's hard to do, but I suppose the only thing to do is make one and find out why it's hard. Honestly I can take a stab at it this evening :)

Reply:
tada: https://hexdocs.pm/input/input.html